services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None,
```

Comment:
str or None annotation. Fix everywhere

Reply:
This haven't been added - you need to add annotation that this parameter can also be None